packages/rest/src/keys.ts

```diff
@@ -50,6 +50,14 @@ export namespace RestBindings {
    * Binding key for setting and injecting the protocol of RestServer
    */
   export const PROTOCOL = BindingKey.create<'http' | 'https'>('rest.protocol');
+  /**
+   * Binding key for location of TLS key
+   */
+  export const KEY = BindingKey.create<string | undefined>('rest.key');
+  /**
+   * Binding key for location of TLS certificate
+   */
+  export const CERT = BindingKey.create<string | undefined>('rest.cert');
```

Comment:
Let's not expose KEY/CERT, at least for now.

1. It's possible to configure HTTPS server using PFX instead of CERT format.
2. I am concerned about security implications, the private keys must be strongly protected to prevent accidental leaks.